docs/management/connectors/action-types/servicenow.asciidoc

```diff
@@ -1,12 +1,69 @@
 [role="xpack"]
 [[servicenow-action-type]]
-=== ServiceNow connector and action
+=== ServiceNow ITSM connector and action
 ++++
 <titleabbrev>ServiceNow ITSM</titleabbrev>
 ++++
 
 The ServiceNow ITSM connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create ServiceNow incidents.
 
+[float]
+[[servicenow-itsm-connector-prerequisites]]
+==== Prerequisites
+After upgrading to 7.16, you must complete the following within your ServiceNow instance before creating a new ServiceNow ITSM connector or <<servicenow-itsm-connector-update, updating an existing one>>:
+
+* Install https://store.servicenow.com/sn_appstore_store.do#!/store/application/7148dbc91bf1f450ced060a7234bcb88[Elastic for ITSM] from the ServiceNow Store.
+* Create a ServiceNow integration user and assign it the appropriate roles.
+* Create a CORS rule.
+
+*Create a ServiceNow integration user*
+
+To ensure authenticated communication between Elastic and ServiceNow, create a ServiceNow integration user and assign it the appropriate roles. 
+
+. Go to *System Security -> Users*.
+. Click *New*.
+. Complete the form.
+. Assign the integration user the following roles: 
+* `import_set_loader`
+* `import_transformer`
+* `personalize_choices`
+* `x_elas2_inc_int.integration_user`
+. Click *Submit* to create the user.
+
+*Create a CORS rule*
+
+A CORS rule is required for communication between the Elastic and ServiceNow. To create a CORS rule:
+
+. Go to *System Web Services -> CORS Rules*.
+. Click *New*.
+. Configure the rule settings to use the Elastic ITSM API and the Kibana URL:
+* *REST API*: Choose `Elastic ITSM API [x_elas2_inc_int/elastic_api]`.
+* *Domain*: Enter the Kibana URL.
+. Go to the *HTTP methods* tab and select *GET*. This chooses the GET HTTP method.
+. Click *Submit* to create the rule.
+
+[float]
+[[servicenow-itsm-connector-update]]
+==== Update a deprecated ServiceNow ITSM connector
+
+ServiceNow ITSM connectors created in 7.15 or earlier are marked as deprecated after you upgrade to 7.16. Deprecated connectors have a yellow icon after their name and display a warning message when you select them.
+
+[role="screenshot"]
+image::management/connectors/images/servicenow-sir-update-connector.png[Shows deprecated ServiceNow connectors]
+
+IMPORTANT: Deprecated connectors will continue to function with the rules they were added to and can be assigned to a new rule. However, we strongly recommend updating the deprecated connector or <<creating-new-connector, creating a new one>> to ensure you have access to connector enhancements, such as updating incidents.
+
+To update a deprecated connector:
+
+. Open the main menu and go to *Stack Management -> Rules and connectors -> Connectors*.
+. Select the deprecated connector to open the connector Edit flyout.
```

Comment:
"Edit connector flyout" instead of "connector Edit flyout"?

Reply:
Good catch, thanks!